app/forms/admin/participants/change_relationship/wizard_steps/change_training_programme_step.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Admin
+  module Participants
+    module ChangeRelationship
+      module WizardSteps
+        class ChangeTrainingProgrammeStep < ::JourneyWizard::Step
+          attr_accessor :selected_partnership
+
+          validates :selected_partnership, inclusion: { in: :permitted_options }
+
+          def self.permitted_params
+            %i[
+              selected_partnership
+            ]
+          end
+
+          def expected?
+            wizard.programme_can_be_changed?
+          end
+
+          def next_step
+            if wizard.create_new_partnership?
+              :choose_lead_provider
+            else
+              :confirm_selected_partnership
+            end
+          end
+
+          def options
+            list = [
+              OpenStruct.new(id: "create_new", name: "<strong>Create new relationship</strong>".html_safe),
+            ]
+            available = wizard.school_partnerships - [wizard.current_partnership]
+            list << available.map do |partnership|
+              OpenStruct.new(id: partnership.id, name: partnership_label(partnership))
+            end
+            list.flatten
+          end
+
+        private
+
+          def permitted_options
+            options.map(&:id)
+          end
+
+          def partnership_label(partnership)
+            label = []
+            label << "<strong>Default partnership:</strong>" unless partnership.relationship?
```

Comment:
No need for bold here.

```suggestion
            label << "Default partnership:" unless partnership.relationship?
```

Reply:
These were just the designs on the ticket, I'm not sure if the previous designer did them or not but happy to change. Do you think this would look better if "Default partnership" was put at the end in brackets perhaps? i.e. "Lead provider name/Delivery partner name (Default partnership)".  Or this could be "(School default)" or something like that? @frankieroberto

Reply:
Having brackets after the name works for me. 👍🏻

Reply:
changed to  `(Default partnership)` as a suffix